script/Arcueid/Day 8/08_00_ARC08_3_2.txt

```diff
@@ -267,7 +293,8 @@ Wouldn't it be better to show some guts for once and accept whatever comes at yo
 [sha:f062ae3cb195cff3fa3ef6d746613286b65cc39a]{
 -- Page 214, Offset 20202.
 -- この通り、餌に釣られて破壊者が追いかけてくれば、窮地は脱したと言っていいだろう。
-As you can see, I can get out of this predicament if I lure the destroyer with my bait and make her chase me.
+If I can act as bait and lure this destroyer into chasing after me,
```

Comment:
"this destructive force" maybe? In the other file it worked a bit better in the sentence